code/wdb/datastripe.q

```diff
@@ -85,12 +85,13 @@ savetablesoverperiod:{[dir;tablename;nextp;lasttime]
     partitionlist: ?[tablename;();();(distinct;keycol)];
 
     /- enumerate and then split by keycol
-    enumkeycol: .Q.en[dir;?[tablename;enlist (<;`time;nextp);0b;()]];
+    symdir:` sv .ds.td,.proc.procname;
```

Comment:
should this be `.ds.td` or `dir`? (I see `dir` is passed to the function & used below in call to `upserttopartition`, just want to make sure this is correct)

Reply:
I just wanted to clarify this - if it was correct as it was, that's fine. I don't know the context enough to know which is correct

Reply:
It should have been dir to ensure that the directory structure is consistent even if you pass in a different directory when running the savedown. I think it's unlikely that this would come up but it did need corrected.